gum/backend-darwin/gumkernel-darwin.c

```diff
@@ -166,6 +167,39 @@ gum_kernel_alloc_n_pages (guint n_pages)
   return result + page_size;
 }
 
+gboolean
+gum_kernel_try_free_pages (GumAddress mem)
+{
+  mach_port_t task;
+  gsize page_size;
+  mach_vm_address_t address;
+  mach_vm_size_t *size;
+  gsize bytes_read;
+  kern_return_t kr;
+
+  task = gum_kernel_get_task ();
+  if (task == MACH_PORT_NULL)
+    return FALSE;
+
+  page_size = vm_kernel_page_size;
+
+  address = mem - page_size;
+  size = (mach_vm_size_t*) gum_kernel_read (address, sizeof (mach_vm_size_t),
```

Comment:
Missing space before `*`:

```suggestion
  size = (mach_vm_size_t *) gum_kernel_read (address, sizeof (mach_vm_size_t),
```